targets/nuttx-stm32f4/nuttx/config_mpu

```diff
@@ -0,0 +1,1673 @@
+#
+# Automatically generated file; DO NOT EDIT.
```

Comment:
The `targets/nuttx-stm32f4/nuttx` directory already contains various config files following the naming pattern of `.config.*`. It would be nice to keep things consistent, IMHO, which would mean a `.config.mpu` name for this file.